config/config/global_promotionconfiguration.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: promoter.argoproj.io/v1alpha1
+kind: PromotionConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/name: promoter
+    app.kubernetes.io/part-of: promoter
+    app.kubernetes.io/managed-by: kustomize
+  name: global
+spec:
+  pullRequest:
+    template:
+      title: "Promote {{ trunc 5 .Status.Proposed.Dry.Sha }} to `{{ .Spec.ActiveBranch }}`"
+      body: "This PR is promoting the environment branch `{{ .Spec.ActiveBranch }}` which is currently on dry sha {{ .Status.Active.Dry.Sha }} to dry sha {{ .Status.Proposed.Dry.Sha }}."
```

I would go for `description` instead of `body`. 

```suggestion
      description: "This PR is promoting the environment branch `{{ .Spec.ActiveBranch }}` which is currently on dry sha {{ .Status.Active.Dry.Sha }} to dry sha {{ .Status.Proposed.Dry.Sha }}."
```